### ISSUE TITLE
Passing date to metrics retrieve does not cast to timestamp correctly.

### ISSUE THREAD

**Issue by JonesAndrew** [delighted/delighted-python#9]
The example

```
metrics = delighted.Metrics.retrieve(since=datetime.date(2013, 10, 01),
                                     until=datetime.date(2013, 11, 01))
```

doesn't correctly convert the dates to the timestamp needed for the request.

**Comment by mkdynamic** [comment]
@JonesAndrew Thanks for the bug report! Can you let me know what Python version you're using?

**Comment by JonesAndrew** [comment]
@mkdynamic I'm using 2.7.11

**Comment by mkdynamic** [comment]
This should be fixed in 2.0.0b1, please check it out and let us know how it goes.
